src/main.js

```diff
@@ -1,11 +1,25 @@
 function copyToClipboard(str) {
   const el = document.createElement('textarea');
   el.value = str;
-  el.setAttribute('readonly', '');
+  el.contentEditable = true;
+  el.readOnly = true;
   el.style.position = 'absolute';
   el.style.left = '-9999px';
+  
   document.body.appendChild(el);
-  el.select();
+
+  if (/iPad|iPhone|iPod/.test(navigator.userAgent)) {
```

Comment:
Esse `if` é necessário? Digo, se esse código específico funcionar nos outros casos, dá pra deixar só ele

Reply:
The iOS hack don't work on Chrome, so the `if` is needed.